source/manifest.json

```diff
@@ -45,6 +45,9 @@
 				"https://github.com/*",
 				"https://gist.github.com/*"
 			],
+			"exclude_matches": [
+				"https://*/login/*"
+			],
```

Comment:
I initially thought about add this pageDetect inside `features.add`, but this is better for such static URLs

```js
const isLogin = (): boolean => location.pathname.split('/')[1] === 'login' || pageDetect.isPasswordConfirmation();
```